DEVELOPMENT.md

```diff
@@ -118,6 +118,22 @@ kubectl create clusterrolebinding cluster-admin-binding \
   --user="${USER}"
 ```
 
+### Install in custom namespace
+1. To install into a different namespace you will need to modify resources in the `./config`
+    - remove all `namespace: tekton` resources
+    - delete the `namespace.yaml
+    - modify the `subjects.namespace` value to the desired namespace
```

Comment:
Again can you link `clusterrolebinding` yaml here? I think thats what you are referring here

Reply:
done